app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -218,12 +197,19 @@ class OverdueScreen : BaseScreen<
 
     buttonsFrame.applyInsetsBottomPadding()
 
-    overdueRecyclerView.adapter = overdueListAdapter
-    overdueRecyclerView.layoutManager = LinearLayoutManager(context)
+    composeView.apply {
+      setViewCompositionStrategy(
+          ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
+      )
+      setContent {
+        val uiModels by uiModelsState
```

Comment:
Once we make the `uiModelsState` as a mutable variable, we don't need to create this variable anymore. We can use it directly.